components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -70,11 +68,50 @@ namespace brave {
 
 namespace {
 
+// NOTE TO READER: These headers are for partners to determine the browser is
+// Brave without creating webcompat issues by creating a new user-agent. Since
+// there are tens of millions of Brave users, this presents no privacy risk.
+// Custom headers are deprecated and new partners use the
+// navigator.brave.isBrave() JavaScript API.
```

Comment:
Would defer to @fmarier on this (is this comment accurate (no privacy risk))

Reply:
This comment is a great idea. This feature has been misunderstood ever since it came out I think.

My suggestion would be to change this sentence:
>  Since there are tens of millions of Brave users, this presents no privacy risk

to this:
> All Brave users send the exact same header and so this cannot be used for tracking individual users. See https://github.com/brave/brave-browser/wiki/Custom-Headers for more information.

That way we simply stick to the facts, which speak for themselves IMHO.